sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -6,10 +6,14 @@
 - Added PhoneNumberAdministrationClient (originally was part of the Azure.Communication.Administration package).
 - Added support to create PhoneNumberAdministrationClient with AzureKeyCredential.
 - Added support to create PhoneNumberAdministrationClient with TokenCredential
+- Added support for long running operations
```

Comment:
"." at the end of line. Please update the other missing ones too.

Reply:
Done.